packages/mdc-select/README.md

```diff
@@ -44,164 +40,117 @@ npm install @material/select
         `with value "${select.value}"`);
 });
 ```
 
 See [Importing the JS component](../../docs/importing-js.md) for more information on how to import JavaScript.
 
-Note that you can include mdc-select via a UMD bundle, which will be available post-alpha.
-
-> Note that the full-fidelity version of MDC Select requires you to manually include the styles for
-[mdc-menu](../mdc-menu) and [mdc-list](../mdc-list). If you are using the
-[material-components-web](../material-components-web) package, this is taken care of for you and you
-simply need to import the main stylesheet. Otherwise, _you must ensure that you manually include the
-style dependencies for both the mdc-list and mdc-menu for this component to function properly._
-
 #### Select with pre-selected option
 
 When dealing with the select component that has pre-selected values, you'll want to ensure that you
 render `mdc-select__label` with the `mdc-select__label--float-above` modifier class and the selected
-option with `aria-selected`. This will ensure that the label moves out of the way of the select's value
-and prevents a Flash Of Un-styled Content (**FOUC**).
+option with the `selected` attribute. This will ensure that the label moves out
+of the way of the select's value and prevents a Flash Of Un-styled Content (**FOUC**).
 
 ```html
-<div class="mdc-select" role="listbox">
-  <div class="mdc-select__surface" tabindex="0">
-    <div class="mdc-select__label mdc-select__label--float-above">Pick a Food Group</div>
-    <div class="mdc-select__selected-text"></div>
-    <div class="mdc-select__bottom-line"></div>
-  </div>
-  <div class="mdc-menu mdc-select__menu">
-    <ul class="mdc-list mdc-menu__items">
-      <li class="mdc-list-item" role="option" tabindex="0">
-        Bread, Cereal, Rice, and Pasta
-      </li>
-      <li class="mdc-list-item" role="option" tabindex="0">
-        Vegetables
-      </li>
-      <li class="mdc-list-item" role="option" tabindex="0">
-        Fruit
-      </li>
-      <li class="mdc-list-item" role="option" aria-selected tabindex="0">
-        Milk, Yogurt, and Cheese
-      </li>
-      <li class="mdc-list-item" role="option" tabindex="0">
-        Meat, Poultry, Fish, Dry Beans, Eggs, and Nuts
-      </li>
-      <li class="mdc-list-item" role="option" tabindex="0">
-        Fats, Oils, and Sweets
-      </li>
-    </ul>
-  </div>
+<div class="mdc-select">
+  <select class="mdc-select__surface">
+    <option value="vegetables">
+      Vegetables
+    </option>
+    <option value="fruit">
+      Fruit
+    </option>
+    <option value="dairy" selected>
+      Milk, Yogurt, and Cheese
+    </option>
+  </select>
+  <div class="mdc-select__label mdc-select__label--float-above">Pick a Food Group</div>
+  <div class="mdc-select__bottom-line"></div>
 </div>
 ```
 
+#### Select with floating label as the placeholder
+
+Use the `<option disabled selected hidden/>` element as a way to display the select having no value.
```

Comment:
I was saying to add float-above if you want a placeholder instead of using the label, but the way I worded it is confusing and this might not be the place so I'll remove. Should I also remove `hidden` from the demos as well?

Reply:
Yes, I think I commented on the line in the demo page too.

Reply:
Also, not sure what you mean regarding wanting a placeholder instead of using the label... the `placeholder` attribute is not a thing for native `select` (since `select` normally always has an option selected)